sympy/printing/latex.py

```diff
@@ -979,10 +979,15 @@ def _print_ceiling(self, expr, exp=None):
             return tex
 
     def _print_log(self, expr, exp=None):
-        if not self._settings["ln_notation"]:
-            tex = r"\log{\left(%s \right)}" % self._print(expr.args[0])
+        args = expr.args
+        if len(args) == 1:
+            if self._settings["ln_notation"]:
+                tex = r"\ln{\left(%s \right)}" % self._print(args[0])
+            else:
+                tex = r"\log{\left(%s \right)}" % self._print(args[0])
```

Comment:
Could use `self._add_parens` here

```suggestion
            if self._settings["ln_notation"]:
                tex = r"\ln{%s}" % self._add_parens(self._print(args[0]))
            else:
                tex = r"\log{%s}" % self._add_parens(self._print(args[0]))
```

Note that you'd need to remove some spaces from the LaTeX output test if you did this.